app/views/conferences/_highlights.haml

```diff
@@ -4,7 +4,7 @@
       Program Highlights
 .row
   .col-md-12
-    .row.row-centered
+    .row.row-centered.shuffle-deck
       - highlights.each do |event|
```

Comment:
Thinking about it. Would it be the same as 

```ruby
@highlights = @conference.highlighted_events.eager_load(:speakers).shuffle
```
in the controller?

Sorry I did not catch this earlier.

Reply:
It's not the same! The highlights are part of the cache key for the partial caching, and changing the order changes the key... it would invalidate the cache on every view!

> Since this is done in JS, the cache of highlighted sessions is preserved, maintaining server-side performance.

Reply:
aight